.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+sudo: required
+dist: trusty
+before_script:
+- npm install -g web-component-tester
```

Comment:
The `-g` isn't needed given line 7.

Reply:
I'm not sure if it's the case any longer, but there was a warning by WCT itself where it was asking for `-g`, I'll try it without.